packages/victory-core/src/victory-util/wrapper.js

```diff
@@ -273,27 +273,63 @@ export default {
       } else {
         data = Data.getData(childProps);
       }
-      const attr = axis === "x" ? "xName" : "yName";
-      return data.map((d) => d[attr]).filter(Boolean);
+      return data.map((d) => ({ x: d.xName, y: d.yName }));
     };
-    return Helpers.reduceChildren(childComponents.slice(0), iteratee);
+
+    const initialMemo = { x: [], y: [] };
+    const combine = (memo, datum) => ({
+      x: datum.x ? memo.x.concat(datum.x) : memo.x,
```

Comment:
I think these should be `datum.x !== undefined ?`

Reply:
Though maybe this is unnecessary since the case where these would be empty strings is a bit contrived.

Reply:
`!== undefined` is better, I'll change that.